src/Qir/Runtime/stdlib/include/qir_stdlib.h

```diff
@@ -0,0 +1,230 @@
+    QIR_SHARED_API void __quantum__rt__string_update_reference_count(QirString*, int32_t); // NOLINT
+
+    // Creates a new string that is the concatenation of the two argument strings.
+    QIR_SHARED_API QirString* __quantum__rt__string_concatenate(QirString*, QirString*); // NOLINT
+
+    // Returns true if the two strings are equal, false otherwise.
+    QIR_SHARED_API bool __quantum__rt__string_equal(QirString*, QirString*); // NOLINT
+
+    // Returns a string representation of the integer.
+    QIR_SHARED_API QirString* __quantum__rt__int_to_string(int64_t); // NOLINT
+
+    // Returns a string representation of the double.
+    QIR_SHARED_API QirString* __quantum__rt__double_to_string(double); // NOLINT
+
+    // Returns a string representation of the Boolean.
+    QIR_SHARED_API QirString* __quantum__rt__bool_to_string(bool); // NOLINT
+
+    // Returns a string representation of the Pauli.
+    QIR_SHARED_API QirString* __quantum__rt__pauli_to_string(PauliId); // NOLINT
+
+    // Returns a pointer to an array that contains a null-terminated sequence of characters
+    // (i.e., a C-string) representing the current value of the string object.
+    QIR_SHARED_API const char* __quantum__rt__string_get_data(QirString* str); // NOLINT
+
+    // Returns the length of the string, in terms of bytes.
+    // http://www.cplusplus.com/reference/string/string/size/
+    QIR_SHARED_API uint32_t __quantum__rt__string_get_length(QirString* str); // NOLINT
+
+    // Allocates space for a tuple requiring the given number of bytes and sets the reference count to 1.
+    QIR_SHARED_API QirTuple* __quantum__rt__tuple_create(int64_t); // NOLINT
+
+    // Adds the given integer value to the reference count for the tuple. Deallocates the tuple if the reference count
+    // becomes 0. The behavior is undefined if the reference count becomes negative.
+    QIR_SHARED_API void __quantum__rt__tuple_update_reference_count(QirTuple*, int32_t); // NOLINT
+
+    // Adds the given integer value to the alias count for the tuple. Fails if the count becomes negative.
+    QIR_SHARED_API void __quantum__rt__tuple_update_alias_count(QirTuple*, int32_t); // NOLINT
+
+    // Creates a shallow copy of the tuple if the user count is larger than 0 or the second argument is `true`.
+    QIR_SHARED_API QirTuple* __quantum__rt__tuple_copy(QirTuple*, bool force); // NOLINT
+
+    // Returns a string representation of the big integer.
+    QIR_SHARED_API QirString* __quantum__rt__bigint_to_string(QirBigInt*); // NOLINT
+
+    // Creates a big integer with the specified initial value.
+    QIR_SHARED_API QirBigInt* __quantum__rt__bigint_create_i64(int64_t); // NOLINT
+
+    // Creates a big integer with the initial value specified by the i8 array. The 0-th element of the array is the
+    // highest-order byte, followed by the first element, etc.
+    QIR_SHARED_API QirBigInt* __quantum__rt__bigint_create_array(int, char*); // NOLINT
```

Comment:
FYI: The `int` param is not documented in the QIR Spec. I filed an [issue](https://github.com/qir-alliance/qir-spec/issues/26) against the QIR Spec.

Reply:
That makes me realize I didn't correctly make this a 32-bit integer type in this signature. It should be `int32_t`.